custom_components/localtuya/pytuya/__init__.py

```diff
@@ -170,51 +202,56 @@ def __init__(
         self.version = protocol_version
         self.dev_type = "type_0a"
         self.dps_to_request = {}
+        self.cipher = AESCipher(self.local_key)
+        self.seqno = 0
 
         self.port = 6668  # default - do not expect caller to pass in
 
-    def __repr__(self):
-        """Return internal string representation of object."""
-        return "%r" % ((self.id, self.address),)  # FIXME can do better than this
-
-    def _send_receive(self, payload):
-        """
-        Send single buffer `payload` and receive a single buffer.
+    def exchange(self, command, data=None):
+        """Send and recive a message, returning response from device."""
+        _LOGGER.debug("Sending command %s (device type: %s", self.dev_type)
```

Comment:
This should be 
`_LOGGER.debug("Sending command '%s' (device type: %s)", command, self.dev_type)`